meta-mender-core/recipes-mender/mender-client/mender-client-cpp.inc

```diff
@@ -87,7 +96,9 @@ FILES:mender-auth += "\
     ${systemd_unitdir}/system/mender-authd.service \
 "
 
-DEPENDS += "lmdb googletest"
+DEPENDS += "lmdb googletest boost libarchive"
+RDEPENDS:mender-auth:append = " boost-filesystem"
```

Comment:
Hmm, is this really all the boost stuff we rely on?

Reply:
Yep!

Remember that `RDEPENDS` is only libraries. The rest of the stuff we use is header-only, a build-time dependency, which is therefore covered by the generic `DEPENDS = "boost"`

Reply:
Alright, sweet!